change/@fluentui-react-datepicker-compat-59a888f8-093f-449c-b49e-4f580894da5e.json

```diff
@@ -1,5 +1,5 @@
 {
-  "type": "prerelease",
+  "type": "none",
```

Comment:
You can change all of these to `minor`/`patch` accordingly instead of `none` so they are put correctly in the changelog, since you're already going to be doing a version bump.